independent-publisher-2/functions.php

```diff
@@ -6,154 +6,138 @@
  */
 
 if ( ! function_exists( 'independent_publisher_2_setup' ) ) :
-	/**
-	 * Sets up theme defaults and registers support for various WordPress features.
+/**
```

Comment:
This looks like just an indentation change. The version in GH looks correct but it's not important.